### PR TITLE
feat: copy for pivot + various improvements

### PR DESCRIPTION
**Pull request by Gowiem**
## Info

* Adds all the new copy from working with copy writer on the services / focus pivot. 
  * Source documents:
    * [Home Page Copy](https://docs.google.com/document/d/1-hgrzA8H8Pbl21QUvmnIVLcBwcWSRqpfgGuUc5VDSio/edit?usp=sharing)
    * [Services Page Copy](https://docs.google.com/document/d/1vVWlEeSh1cUBEiboh3tpq4l-e9I8U2aLTR4UoW7Xkok/edit?usp=sharing)
  * Everything that wasn't in those docs (Maybe 10% or 20%), I came up with fresh and should get some extra attention as I'm sure I messed some things up. 
* Adds a new thank you page for subscribing to the newsletter.
* Reworks my bio + adds links to recent podcasts I've been on
* Adds new logos for recent clients

### CODE REVIEW

File: assets/css/custom.scss [hunk 1]
```diff
@@ -2167,7 +2171,7 @@ footer {
     }
   }
 }
-#masterpoint-blueprint-framework {
+#team-training {
   padding: 7.3rem 0 7.8rem;
   color: #fff;
   background: #031e1f;
```
Comment (oycyc) [inline assets/css/custom.scss:2177]:
```suggestion
  background: #031e1f;
  h3 {
    color: #fff
  }
```
This section's background defaults to the green but is hard to see with the background:
image
The other section sets it to white using this same selector
image

**Comment by oycyc** [comment]
BTW did the subscribe link break? I remember signing up for it a while back and it worked fine, but now trying it (not in the preview, in prod www.masterpoint.io , the redirect link is broken and can't tell if the subscription actually went through

**Comment by gberenice** [comment]
I feel like we should structure the service description better and provide more info on them on the service page https://deploy-preview-27--masterpoint.netlify.app/services/
Now it's very similar to what we say on the main page but with the `Get Started` button.
Maybe it was the point - to keep the description simple, then ignore me :)

File: content/thank-you/subscribing.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: Thanks for subscribing!
+description: >-
+   We're looking forward to delivering you awesome, actionable insights about IaC and we know you're going to love them. If you're eager to check out some of our past content, you can <a href="https://masterpoint.beehiiv.com/" target="_blank">access our newsletter archive here</a>.
+   </br></br>
```
Comment (westonplatter) [inline content/thank-you/subscribing.md:5]:
```suggestion
  <br/><br/>
```

File: content/thank-you/subscribing.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+   Reach out if you ever have any feedback or questions @ <a href="mailto:matt@masterpoint.io">matt@masterpoint.io</a>.
+   </br></br>
```
Comment (westonplatter) [inline content/thank-you/subscribing.md:7]:
```suggestion
    <br/><br/>
```

**Comment by westonplatter** [comment]
Jotting down notes
- Services, FAQ 2 - `Our IaC tool of choice is HashiCorp Terraform, which is the leading tool in the industry.` - do we want specify `hashicorp` when talking about tf? I'm thinking it would be better to keep the door open to using either OpenTofu or Hashicorp and position ourselves as experts in working with OpenTofu

File: assets/css/custom.scss (hunk 1 above)
Comment (Gowiem) [reply assets/css/custom.scss:2177]: Great find @oycyc -- Thanks for this. Will be pushed shortly!

**Comment by Gowiem** [comment]
> I feel like we should structure the service description better and provide more info on them on the service page https://deploy-preview-27--masterpoint.netlify.app/services/
> Now it's very similar to what we say on the main page but with the Get Started button.
> Maybe it was the point - to keep the description simple, then ignore me :)

Yeah, this is basically the intention... for better or worse 🤷

File: content/thank-you/subscribing.md (hunk 2 above)
Comment (Gowiem) [reply content/thank-you/subscribing.md:5]: Dealing with in next push!

File: content/thank-you/subscribing.md (hunk 3 above)
Comment (Gowiem) [reply content/thank-you/subscribing.md:7]: Dealing with in next push!

**Comment by Gowiem** [comment]
@westonplatter 

> Services, FAQ 2 - Our IaC tool of choice is HashiCorp Terraform, which is the leading tool in the industry. - do we want specify hashicorp when talking about tf? I'm thinking it would be better to keep the door open to using either OpenTofu or Hashicorp and position ourselves as experts in working with OpenTofu

I forgot about the updates to the FAQs! Check them out in the next push 👍

**Comment by Gowiem** [comment]
@gberenice @oycyc @westonplatter -- Ready for a 2nd review!

FYI, in our FAQ it does say that "we’re cloud agnostic when it comes to IaC delivery and will work with AWS, Azure, and Google Cloud", but I do want to reassure everyone: We're never working on Azure 😎

**Review by gberenice — APPROVED** [review]
LGTM! 🚢

**Review by oycyc — APPROVED** [review]

**Pull request merged**